test/mcp-test-tool.js

```
@@ -0,0 +1,149 @@
+    try {
+      await client.invokeTool(test.tool, test.params);
+      if (test.expectError) {
+        console.warn(`\u26a0\ufe0f ${test.tool} expected error but succeeded`);
+        failedTests++;
+      } else {
+        console.log(`\u2713 ${test.tool} tool OK`);
+        passedTests++;
+      }
+    } catch (error) {
+      if (test.expectError && test.expectError.test(error.message)) {
+        console.log(`\u2713 ${test.tool} error handling OK`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)